src/main/java/internet/shop/Demo.java

```diff
@@ -1,121 +1,152 @@
 package internet.shop;
 
 import internet.shop.lib.Injector;
+import internet.shop.model.Car;
+import internet.shop.model.Driver;
 import internet.shop.model.Manufacturer;
+import internet.shop.service.CarService;
+import internet.shop.service.DriverService;
 import internet.shop.service.ManufacturerService;
 
 public class Demo {
     private static Injector injector = Injector.getInstance("internet.shop");
 
     public static void main(String[] args) {
-        Manufacturer manufacturer = new Manufacturer();
-        manufacturer.setId(10L);
-        manufacturer.setCountry("Ukraine");
-        manufacturer.setName("MiksMaks");
+        Driver driver1 = new Driver();
+        driver1.setId(1L);
+        driver1.setName("Bob");
+        driver1.setLicenceNumber("123-321");
+
+        Driver driver2 = new Driver();
+        driver2.setId(2L);
+        driver2.setName("Alice");
+        driver2.setLicenceNumber("4821-00");
+
+        Driver driver3 = new Driver();
+        driver3.setId(100L);
+        driver3.setName("John");
+        driver3.setLicenceNumber("998-12-33");
+
+        DriverService driverService = (DriverService) injector
+                .getInstance(DriverService.class);
+
+        driverService.create(driver1);
+        driverService.create(driver2);
+        driverService.create(driver3);
+
+        System.out.println(driverService.get(3L));
```

Comment:
```suggestion
        System.out.println(driverService.get(driver3.getId()));
```
you can replaces hardcoded values with `getId()` methods

Reply:
done